### PR TITLE
test with elm-test in buck2

### PR DESCRIPTION
**Pull request by BrianHicks**
This tests the package with elm-test in buck2. It temporarily adds elm-test (the package) to Node's package set, just because it probably doesn't make sense to figure out how to grab binaries from Nix at the same time (but WAT-600 will do that.)

### CODE REVIEW

File: BUCK [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
     elm_json = "elm.json",
     src = "src",
+    tests = [
+        ":elm_test",
+    ],
+)
+
+elm_test(
+    name = "elm_test",
+    test_srcs = glob(["test/**/*.elm"]),
```
Comment (caiquefigueiredo) [inline BUCK:16]:
```suggestion
    test_srcs = glob(["tests/**/*.elm"]),
```

**Review by caiquefigueiredo — APPROVED** [review]

**Comment by BrianHicks** [comment]
@NoRedInkBot r=caiquefigueiredo

**Comment by BrianHicks** [comment]
yay, it worked! Ready for re-review @caiquefigueiredo!

File: script/puppeteer-tests.js [hunk 2]
```diff
@@ -33,6 +46,12 @@ describe("UI tests", function () {
     server.close();
   });
 
+  const handlePageErrors = function (page) {
+    page.on("pageerror", (err) => {
```
Comment (caiquefigueiredo) [inline script/puppeteer-tests.js:50]: should we exit with error when this happens?

**Review by caiquefigueiredo — APPROVED** [review]

File: script/puppeteer-tests.js (hunk 2 above)
Comment (BrianHicks) [reply script/puppeteer-tests.js:50]: I think it’s OK to not for now.

**Pull request merged**